src/components/UsefulContacts/index.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {
+  Hero,
+  SearchResultListItem,
+  SocialsShare
+} from "@code4ro/taskforce-fe-components";
+import contacts from "../../data/useful-contacts";
+
+import "./styles.scss";
+
+const UsefulContacts = () => {
+  return (
+    <>
+      <Hero title="Contacte utile" />
+      <SocialsShare currentPage="https://diasporahub.ro/contacte-utile" />
+
+      {contacts.map((contact, index) => (
+        <div key={index} className="contact-wrapper ">
```

Comment:
nit: We should use some prefixes to avoid key collision on the page, later on. Something like this:
```suggestion
        <div key={`contact_item_${index}`} className="contact-wrapper">
```